appkit/authentication/siwx/default.mdx

```diff
@@ -8,42 +8,36 @@ The **Sign In With X** feature enables decentralized applications (Dapps) to aut
 This feature allows developers using our SDK to implement authentication by having users sign a unique string message with their blockchain wallets. The **Sign In With X** feature is designed in accordance with the [CAIP-122](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-122.md) standard, which establishes a chain-agnostic framework for blockchain-based authentication and authorization on off-chain services.
 
 <Frame caption="SIWX Demo">
-  <video
-    controls
-    autoPlay
-    loop
-    src="/images/siwx.mp4"
-  ></video>
+  <video controls autoPlay loop src="/images/siwx.mp4"></video>
 </Frame>
 
 ## Demo
 
 <CardGroup cols={2}>
-    <Card icon="flask" title="Try Demo" href="https://appkit-lab.reown.com/library/siwx-default/" />
+  <Card icon="flask" title="Try Demo" href="https://appkit-lab.reown.com/library/siwx-default/" />
 </CardGroup>
 
-
 ## Getting Started
 
 **SIWX** works as a plugin system for AppKit and you are going to add the plugin in the AppKit configuration. There are some ways to implement the **SIWX** feature:
 
-- Use the default implementation provided by AppKit
-- Use Cloud Auth SIWX to manage the sessions in the Cloud Dashboard
+- Use Reown Authentication to manage the sessions in the Cloud Dashboard
+- Use the `SIWXDefault` class provided by AppKit
 - Create a custom implementation to suit your specific requirements.
 
 To initialize the **SIWX** feature, you need to add the `siwx` parameter to the `createAppKit` function.
 
-### Default Implementation
+### Reown Authentication
 
-By using the default implementation, you can quickly integrate the **SIWX** feature into your Dapp. The default implementation provides a set of pre-built components that allow you to have the feature up and running in no time.
+Reown Authentication is a predefined implementation of the SIWX configuration plugin that uses the Cloud service to create and manage SIWX messages and sessions. With Reown Authentication, you will be able to see and control the sessions of your users using the [Reown Authentication Dashboard](/cloud/user-management).
```

Comment:
I'd start with something very straight forward, e.g:
Reown Authentication is a SIWX server hosted by Reown in order to speed up the development of dapps requiring a more traditional authentication system and reduce the maintenance burden of such a server.

Reply:
Just saw the copy on https://github.com/reown-com/reown-docs/blob/f14657f5a160192de0888be6ec09d8c72a4e6a06/cloud/reown-authentication.mdx, the intro there looks perfect